distributed/comm/tests/test_ucx.py

```diff
@@ -0,0 +1,313 @@
+import asyncio
+import itertools
+
+import pytest
+
+ucp = pytest.importorskip("ucp")
+
+from distributed import Client
+from distributed.comm import ucx, listen, connect
+from distributed.comm.registry import backends, get_backend
+from distributed.comm import ucx, parse_address
+from distributed.protocol import to_serialize
+from distributed.deploy.local import LocalCluster
+from distributed.utils_test import gen_test, loop, inc  # noqa: 401
+
+from .test_comms import check_deserialize
+
+
+HOST = ucp.get_address()
+
+# Currently having some issues with re-using ports.
+# Tests just hang. Still debugging.
+port_counter = itertools.count(13337)
```

Comment:
I'm not sure, but this *may* be fixed by https://github.com/Akshay-Venkatesh/ucx-py/issues/79. If so we could remove this and just provide the host without the port, and let ucx provide a port for us.